packages/@react-stately/utils/test/useControlledState.test.js

```diff
@@ -270,7 +270,7 @@ describe('useControlledState tests', function () {
     }
 
     function TransitionButton({onClick}) {
-      let [isPending, startTransition] = React.useTransition();
+      let [UNSTABLE_isPending, startTransition] = React.useTransition();
```

Comment:
I don't think you meant to edit this file

Reply:
Oops. I searched to hard. Thanks for catching that.